### PR TITLE
Don't test or build Ubuntu 14 packages

### PR DESCRIPTION
**Pull request by blag**
Waiting on StackStorm/st2cd#428 and StackStorm/st2ci#171 to be merged first.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -232,7 +232,7 @@ jobs:
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
     environment:
-      - DISTROS: "trusty xenial bionic el6 el7"
+      - DISTROS: "xenial bionic el6 el7"
```
Comment (armab) [inline .circleci/config.yml]: This uncovered oversight in previous EL8 work.

Deployment should include `el8` as well:
```suggestion
      - DISTROS: "xenial bionic el6 el7 el8"
```

Comment (blag) [reply .circleci/config.yml]: I noticed that, but that's outside the scope of this PR.

Comment (armab) [reply .circleci/config.yml]: cc @punkrokk

Comment (punkrokk) [reply .circleci/config.yml]: Is there another PR then or do I need to make one?

Comment (armab) [reply .circleci/config.yml]: There is no PR for this kind of fix yet

Comment (blag) [reply .circleci/config.yml:235]: Captured in #4899.

**Pull request merged**